Documentation/Home/SystemExtensions.rst

```diff
@@ -394,6 +395,27 @@ part of the core and extracted at some point in time.
             - `12.4 <https://docs.typo3.org/c/typo3/cms-seo/12.4/en-us/>`__
             - `11.5 <https://docs.typo3.org/c/typo3/cms-seo/11.5/en-us/>`__
 
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: `System Notes <https://docs.typo3.org/c/typo3/cms-sys-note/main/en-us/>`_
+
+         .. container:: card-body
+
+            Allows to add notes to pages, that can only be seen in the backend.
```

Comment:
Maybe the text from the introduction is better:
https://docs.typo3.org/c/typo3/cms-sys-note/main/en-us/Editor/Index.html

"The internal notes are a useful feature for adding context and notes to pages. It provides a way for users to document important information related to specific pages."